app/src/lib/shells/darwin.ts

```diff
@@ -22,43 +22,52 @@ export function parse(label: string): Shell {
   return parseEnumValue(Shell, label) ?? Default
 }
 
-function getBundleID(shell: Shell): string {
+function getBundleIDs(shell: Shell): string[] {
   switch (shell) {
     case Shell.Terminal:
-      return 'com.apple.Terminal'
+      return ['com.apple.Terminal']
     case Shell.iTerm2:
-      return 'com.googlecode.iterm2'
+      return ['com.googlecode.iterm2']
     case Shell.Hyper:
-      return 'co.zeit.hyper'
+      return ['co.zeit.hyper']
     case Shell.PowerShellCore:
-      return 'com.microsoft.powershell'
+      return ['com.microsoft.powershell']
     case Shell.Kitty:
-      return 'net.kovidgoyal.kitty'
+      return ['net.kovidgoyal.kitty']
     case Shell.Alacritty:
-      return 'org.alacritty'
+      return ['org.alacritty', 'io.alacritty']
     case Shell.Tabby:
-      return 'org.tabby'
+      return ['org.tabby']
     case Shell.WezTerm:
-      return 'com.github.wez.wezterm'
+      return ['com.github.wez.wezterm']
     case Shell.Warp:
-      return 'dev.warp.Warp-Stable'
+      return ['dev.warp.Warp-Stable']
     default:
       return assertNever(shell, `Unknown shell: ${shell}`)
   }
 }
 
-async function getShellPath(shell: Shell): Promise<string | null> {
-  const bundleId = getBundleID(shell)
-  try {
-    return await appPath(bundleId)
-  } catch (e) {
-    // `appPath` will raise an error if it cannot find the program.
-    return null
+async function getShellPath(
```

Comment:
Given how this is not only the path now, I'd probably name it more generically:
```suggestion
async function getShellInfo(
```

Reply:
Fixed